playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -37,3 +37,15 @@ deployment_vars:
     security_groups: [ 'public' ]
     vpc_subnet:
     assign_public_ip:
+  openshift-enterprise:
```

Comment:
I'm wondering if we should go ahead and create a new variable that can be used for setting multiple deployment types here to avoid duplication.
Something like:
```
rhel7_ent_base:
    # rhel-7.1, requires cloud access subscription
    image: ami-10663b78
    image_name:
    region: us-east-1
    ssh_user: ec2-user
    sudo: yes
    keypair: libra
    type: m4.large
    security_groups: [ 'public' ]
    vpc_subnet:
    assign_public_ip:

enterprise: "{{ rhel7_ent_base }}"
openshift-enterprise: "{{ rhel7_ent_base }}"
atomic-enterprise: "{{ rhel7_ent_base }}"
```